move-vm-backend/src/lib.rs

```diff
@@ -129,17 +129,14 @@ where
         &self,
         address: AccountAddress,
         name: &str,
-    ) -> Result<Option<Vec<u8>>, Error> {
+    ) -> Result<Option<ModuleAbi>, Error> {
```

Comment:
Let's move ModuleAbi and the whole abi.rs to `move-vm-backend-common` so we can import it in `smove` and upper substrate layer without all the heavy stuff in `move-vm-backend`

Reply:
Done